package.json

```diff
@@ -18,32 +18,38 @@
     "node": ">=0.10.0"
   },
   "scripts": {
-    "test": "mocha",
-    "prepublish": "browserify -o browser.js -e index.js"
```

Comment:
Without this, how do you enforce that the built asset is up to date when publishing?